src/api/ledger/transaction.js

```diff
@@ -89,11 +89,20 @@ function getTransactionAsync(identifier: string, options: TransactionOptions,
     }
   }
 
+
+  function maxLedgerGetter(error_?: Error, tx?: Object) {
+    remote.getLedgerSequence(function(err?, seq: number) {
```

Comment:
Since this is in "src/api", we should consistently use "ledgerVersion" instead of "seq"/"sequence"/"ledgerSequence"/"sequence" (here and below)